tests/cypress/integration/filter.spec.js

```diff
@@ -0,0 +1,269 @@
+describe('app', () => {
+  // Rhino applications have an 'app' prefix
+  const nsPrefix = 'app-';
+  // Support function to help generating namespaces
+  const ns = (id) => `#${nsPrefix}${id}`;
+  // Support function to help generating teal namespaces on main UI
+  const nsTeal = (id) => ns(`teal_wrapper-teal-main_ui-${id}`);
+
+  beforeEach(() => {
+    cy.visit('/');
+
+    cy
+      .get('html')
+      .not('.shiny-busy');
+
+    // Find the tab and click to navigate
+    cy
+      .get('.nav.nav-pills a[data-bs-toggle=tab]')
+      .contains('KM plot for TTDE')
```

Comment:
It seems weird to do this only for 1 tab. It might also require an additional Node.js package.

I'm game to do it for all tabs, but only if it is possible to read the yml file (which I'm not 100% sure we can do right now in cypress2 tests)